akka-typed/src/main/java/akka/typed/javadsl/Actor.java

```diff
@@ -371,4 +371,28 @@ else if (Behavior.isAlive(behv))
     return new Deferred<T>(producer);
   }
 
+  // FIXME docs
+  static public <T> Behavior<T> mutable(akka.japi.function.Function<ActorContext<T>, MutableBehavior<T>> producer) {
+    return deferred(ctx -> producer.apply(ctx));
+  }
+
+  //FIXME docs
+  static public abstract class MutableBehavior<T> extends ExtensibleBehavior<T> {
```

Comment:
Why not directly use ExtensibleBehavior? This class adds very little, which makes me doubt whether it pulls its weight.